nemo/collections/asr/parts/utils/activations.py

```diff
@@ -15,7 +15,7 @@
 import torch
 import torch.nn as nn
 
-__all__ = ['Swish', 'Snake']
+__all__ = ['Swish', 'Snake', 'HalfSnake']
```

Comment:
Wondering if we should move Snake and HalfSnake to audio collection instead of asr

Reply:
If I add it to common, should it be in the `activation_registry`? The registry only seems useful if the activation does not require input arguments.

Otherwise, it seems roundabout to have something like:

```
if activation in ["snake", "half_snake"]:
  self.activation = activation_registry[activation](channels)
else:
  self.activation = activation_registry[activation]()
```